setup.py

```diff
@@ -29,14 +29,15 @@ def find_version(filename: str = "version") -> str:
         ).decode().strip()
         build: List[str] = gitversion.split("-")
         # <tagname>-<ncommits-ahead>-<commit-id> (e.g. 0.2-8-adfebee)
-        if len(build) > 1:
-            return "{}.post{}".format(build[0], build[1])
-
-        # tagged commit
-        return gitversion
+        return "{}.post{}".format(build[0], build[1])
```

Comment:
wouldn't this return version with `post` post-fix even for cleanly tagged release versions (like `1.3`)